coverage_comment/subprocess.py

```diff
@@ -25,9 +26,11 @@ def run(*args, **kwargs) -> str:
 
 
 class Git:
-    cwd = ""
+    cwd = "."
 
     def _git(self, *args, **kwargs):
+        if env := kwargs.pop("env", None):
+            kwargs["env"] = os.environ | env
```

Comment:
Because with subprocess, either we don't define an `env` param and the current process environment is inherited, or we define one and it entirely replaces the env. I don't really want to not pass the env (like... I'm perfectly ok that the subprocess receives a PATH variable without having to define one) so I merge the variables I want to add with the environment before passing it to subprocess.

Does it make sense ? You're probably right that a comment should make this clear !

Reply:
Yes, that makes sense, thank you very much.
But then couldn't we just always pass `os.environ` and merge the passed env into it if necessary? For example like this: 
```suggestion
        kwargs["env"] = os.environ | kwargs.pop("env", {})
```
This would explicitly use `os.environ` in case no env was passed, but the subprocess inherits that anyway. I don't know what you're thinking, but I'd find this easier to understand because I was just wondering that in the case where no env was passed, `os.environ` doesn't seem to be used (because I didn't think of it being inherited). It would only represent explicitly what happens implicitly, right?

Reply:
Yeah, I like relying on default behaviours when applicable, but this is a case where doing so hinders readability a lot, so you're right.

Reply:
> python -m this | grep Readability

Yes, I also had to think of the Zen of Python when I wrote the comment :D